pages/configuration-files/[...id].js

```diff
@@ -145,9 +149,24 @@ export async function getServerSideProps({ params, req, query, resolvedUrl }) {
   );
   if (FetchRequest.isResponseSuccess(configurationFile)) {
     const fileSet = await request.getObject(configurationFile.file_set, null);
-    const seqSpecOf = configurationFile.seqspec_of
-      ? await requestFiles(configurationFile.seqspec_of, request)
-      : [];
+
+    // Get any seqspec_of files, as well as their associated seqspec files.
+    let seqspecOf = [];
+    let seqspecFiles = [];
+    if (configurationFile.seqspec_of) {
+      seqspecOf = await requestFiles(configurationFile.seqspec_of, request);
+
+      if (seqspecOf.length > 0) {
+        const seqspecPaths = seqspecOf
```

Comment:
i'm not sure of seqspecPaths. seqspec_of is reverse linked to file.seqspec so I think seqspec_of.file.seqspec should give the same results? Talked to Forrest to drop Associated Seqspec File from 'seqspec File of' table